### PR TITLE
Adding credit checkout page and analytics information into the readme…

### PR DESCRIPTION
**Pull request by awais786**
Updating readme file. 
Adding information about segment key information.
Adding information about waffle switch for the new custom credit checkout page.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -73,10 +73,26 @@ switches exist:
...
+---------
+
+To use google analytics for specific events e.g button clicks you need to add the segment key into the settings file.e.g
+SEGMENT_KEY = 'your segment key'
+
+
+Credit
+------
+
+To enable custom credit checkout page please add the following waffle switch.
```
Comment (zubair-arbi) [inline README.rst]: Nitpick: `To enable custom credit checkout page, please add the following waffle switch:`

File: README.rst [hunk 2]
```diff
@@ -73,10 +73,26 @@ switches exist:
 +--------------------------------+---------------------------------------------------------------------------+
 | publish_course_modes_to_lms    | Publish prices and SKUs to the LMS after every course modification        |
 +--------------------------------+---------------------------------------------------------------------------+
+| ENABLE_CREDIT_APP              | Enable the credit checkout page from where student can purchase credit    |
```
Comment (zubair-arbi) [inline README.rst]: nitpick: `student` -> `student's`

File: README.rst [hunk 3]
```diff
@@ -73,10 +73,26 @@ switches exist:
...
+|                                | courses.                                                                  |
++--------------------------------+---------------------------------------------------------------------------+
 
 .. _Waffle: https://waffle.readthedocs.org/
 
 
+Analytics
+---------
+
+To use google analytics for specific events e.g button clicks you need to add the segment key into the settings file.e.g
```
Comment (zubair-arbi) [inline README.rst]: Change this line to:

```
To use google analytics for specific events e.g., button clicks, you need to add the segment key into the settings file:
```

**Comment by zubair-arbi** [comment]
Small nitpicks, otherwise :+1:

**Comment by clintonb** [comment]
:+1:

**Pull request merged**